sql/wsrep_sst.cc

```diff
@@ -190,6 +190,7 @@ static void wsrep_sst_complete (THD*                thd,
 {
   Wsrep_client_service client_service(thd, thd->wsrep_cs());
   Wsrep_server_state::instance().sst_received(client_service, rcode);
+  wsrep_init_gtid();
```

Comment:
This point is too late to initialize wsrep GTID state. Call to `sst_received()` above will give applier threads permission to proceed, and they may access global GTID state before `wsrep_init_gtid()` gets called.